test/controllers/declaration/AdditionalFiscalReferencesControllerSpec.scala

```diff
@@ -47,11 +49,15 @@ class AdditionalFiscalReferencesControllerSpec extends CustomExportsBaseSpec {
       val Some(result) = route(app, getRequest(uri))
 
       status(result) must be(OK)
+      verify(mockExportsCacheService).getItemByIdAndSession(any[String], any[String])
```

Comment:
In `org.mockito.ArgumentMatchers` you can find method `anyString` it's probably better to use instead of `any[String]`

Reply:
Tbh I don't think theres much value in verifying stuff like that anyway, if we didn't call it the test wouldn't work

Reply:
I'll remove it